src/libs/OptionsListUtils.ts

```diff
@@ -1333,20 +1333,20 @@ function getShareLogOptions(options: OptionList, betas: Beta[] = []): Options {
  * Build the IOUConfirmation options for showing the payee personalDetail
  */
 function getIOUConfirmationOptionsFromPayeePersonalDetail(personalDetail: OnyxEntry<PersonalDetails>, amountText?: string): PayeePersonalDetails {
-    const formattedLogin = LocalePhoneNumber.formatPhoneNumber(personalDetail?.login ?? '');
+    const login = personalDetail?.login ?? '';
     return {
-        text: PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, formattedLogin),
-        alternateText: formattedLogin || PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, '', false),
+        text: LocalePhoneNumber.formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, login)),
+        alternateText: LocalePhoneNumber.formatPhoneNumber(login || PersonalDetailsUtils.getDisplayNameOrDefault(personalDetail, '', false)),
         icons: [
             {
                 source: personalDetail?.avatar ?? FallbackAvatar,
-                name: personalDetail?.login ?? '',
+                name: LocalePhoneNumber.formatPhoneNumber(personalDetail?.login ?? ''),
                 type: CONST.ICON_TYPE_AVATAR,
                 id: personalDetail?.accountID,
             },
         ],
         descriptiveText: amountText ?? '',
-        login: personalDetail?.login ?? '',
+        login: LocalePhoneNumber.formatPhoneNumber(personalDetail?.login ?? ''),
```

Comment:
Please avoid updating the login here. Instead, we will only update the `displayName` where it is used to ensure that we do not introduce any regressions.

Reply:
ok, i did a test on spliting expenses and i agree. It will be good if you can also confirm by a visual test